strongbox-web-core/pom.xml

```diff
@@ -113,6 +113,9 @@
                         <strongbox.basedir>${project.build.directory}</strongbox.basedir>
                         <logging.config>${project.build.directory}/strongbox/etc/logback-spring.xml</logging.config>
                         <logging.debug>true</logging.debug>
+                        <strongbox.npm.remote.changes.enabled>false</strongbox.npm.remote.changes.enabled>
+                        <strongbox.nuget.download.feed>false</strongbox.nuget.download.feed>
+                        <strongbox.download.indexes>false</strongbox.download.indexes>
```

Comment:
You should also disabled them in the distribution as well.

Reply:
For distribution I am thinking to disable it from `setenv.conf `. Am I on right path @steve-todorov ?

Reply:
Yes, that's correct. Better to have them disabled for now in the distribution as well. :)

Reply:
> For distribution I am thinking to disable it from setenv.conf . Am I on right path @steve-todorov ?

right, as I can see it disabled